src/plugin/file/kml/kmlparser.js

```diff
@@ -1396,41 +1392,42 @@ plugin.file.kml.KMLParser.reduceStyles_ = function(config, set) {
  * @private
  */
 plugin.file.kml.KMLParser.prototype.applyStyles_ = function(el, feature) {
-  var styleSets = [];
+  var styles = [];
   var highlightStyle = null;
 
   // style from style url
   var styleUrl = /** @type {string} */ (feature.get('styleUrl'));
   if (styleUrl) {
     var styleId = this.getStyleId(decodeURIComponent(styleUrl));
-    styleSets.push(styleId in this.styleMap_ ? this.styleMap_[styleId] : null);
+    styles.push(styleId in this.styleMap_ ? this.styleMap_[styleId] : null);
     highlightStyle = styleId in this.highlightStyleMap_ ? this.highlightStyleMap_[styleId] : null;
```

Comment:
With the `readStyle` change, `this.highlightStyleMap_` will now be `<string, Object>`. That needs to be updated in the JSDoc and accounted for when setting the custom highlight on the feature.

Reply:
I'm surprised the compiler didn't complain about that.